.tekton/basic-component-pipeline.yaml

```diff
@@ -356,6 +356,31 @@ spec:
       operator: in
       values: [ "false" ]
 
+  - name: roxctl-image-scan
```

Comment:
[nit about the name] Why not `acs-image-scan`? I think, the usage of `roxctl` is just an implementation detail.